spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/GlobalAveragePooling3D.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.keras
+
+import com.intel.analytics.bigdl.nn.abstractnn._
+import com.intel.analytics.bigdl.nn._
+import com.intel.analytics.bigdl.nn.VolumetricAveragePooling
+import com.intel.analytics.bigdl.nn.{Sequential => TSequential}
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Shape
+
+import scala.reflect.ClassTag
+
+class GlobalAveragePooling3D[T: ClassTag](dataFormat: String = "CHANNEL_FIRST",
```

Comment:
Only support channel_first, so can remove this parameter... and write it in the documentation...